### PR TITLE
add Ardour4

### PR DESCRIPTION
**Pull request by magnetophon**
These changes look bigger in the diff than they are: I renamed the ardour dir to ardour3 and made some small changes to it to create ardour4.

### CODE REVIEW

File: pkgs/applications/audio/ardour/ardour3.nix [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    Type=Application
+    X-MultipleArgs=false
+    Categories=GTK;Audio;AudioVideoEditing;AudioVideo;Video;
+    EOF
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Multi-track hard disk recording software";
+    longDescription = ''
+      Also read "The importance of Paying Something" on their homepage, please!
```
Comment (jagajaga) [inline pkgs/applications/audio/ardour/ardour3.nix]: That seems to be not a description..

**Comment by jagajaga** [comment]
@cillianderoiste you ok with that `longDescription`?

**Comment by cillianderoiste** [comment]
@jagajaga I agree it should be improved, but it isn't a change which was introduced in these commits. I've tested Ardour 4 locally, and I think it's good to have it. The longDescription can always be fixed later.

**Comment by jagajaga** [comment]
@cillianderoiste That won't be changed in a near future, I'm sure. Tha's why it was a good idea to change it now..

**Comment by cillianderoiste** [comment]
@jagajaga probably, but personally I don't like to reject changes which improve things and don't make things any worse than they were before.

**Comment by jagajaga** [comment]
@cillianderoiste I'm sure that @magnetophon can fix this fast. Because I'm not rejecting this PR, I was asking to improve it.

**Comment by cillianderoiste** [comment]
Fair point, I'm working on adding support for some extra features and tidying up the expression a bit at the moment anyway so I'll add a better longDescription. Thanks for bringing it up.

**Pull request merged**